public/components/query_compare/home.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect } from 'react';
+import { NavigationPublicPluginStart } from 'src/plugins/navigation/public';
+import { CoreStart, ChromeBreadcrumb } from '../../../../../src/core/public';
+import '../../ace-themes/sql_console';
+import { CreateIndex } from './create_index';
+import { SearchResult } from './search_result';
+import { useSearchRelevanceContext } from '../../contexts';
+import { DocumentsIndex } from '../../types/index';
+import { ServiceEndpoints } from '../../constants';
+
+import './home.scss';
+
+interface QueryExplorerProps {
+  parentBreadCrumbs: ChromeBreadcrumb[];
+  notifications: CoreStart['notifications'];
+  http: CoreStart['http'];
+  navigation: NavigationPublicPluginStart;
+  setBreadcrumbs: (newBreadcrumbs: ChromeBreadcrumb[]) => void;
+  setToast: (title: string, color?: string, text?: any, side?: string) => void;
+  chrome: CoreStart['chrome'];
+}
+
+export const Home = ({
+  parentBreadCrumbs,
+  notifications,
+  http,
+  navigation,
+  setBreadcrumbs,
+  setToast,
+  chrome,
+}: QueryExplorerProps) => {
+  const { documentsIndexes, setDocumentsIndexes } = useSearchRelevanceContext();
+
+  useEffect(() => {
+    setBreadcrumbs([...parentBreadCrumbs]);
+  }, [setBreadcrumbs, parentBreadCrumbs]);
+
+  // Get Indexes
+  useEffect(() => {
+    http.get(ServiceEndpoints.GetIndexes).then((res: DocumentsIndex[]) => {
+      setDocumentsIndexes(res);
+    });
+  }, [http, setDocumentsIndexes]);
```

Comment:
I think the implementation here should be: 
```
  useEffect(() => {
    http.get(ServiceEndpoints.GetIndexes).then((res: DocumentsIndex[]) => {
      setDocumentsIndexes(res);
    });
  }, []);
```
Can remove `http`& `setDocumentsIndexes` from this useEffect snippet. Any particular reason you added this here? In my understanding we need to make this call everytime the page is loaded.

Reply:
I think we will still trigger this call with `http` & `setDocumentsIndexes` every time the page is loaded. I added it to follow the ESLint rule: https://github.com/facebook/react/issues/14920

Reply:
Resolving comment as per @ps48's thumbs up.